special-pages/pages/duckplayer/app/index.js

```diff
@@ -55,10 +58,18 @@ export async function init(messaging, telemetry, baseEnvironment) {
         .withFeatureState('pip', init.settings.pip)
         .withFeatureState('autoplay', init.settings.autoplay)
         .withFeatureState('focusMode', init.settings.focusMode)
-        .withDisabledFocusMode(baseEnvironment.urlParams.get('focusMode'));
+        .withFeatureState('customError', init.settings.customError)
+        .withDisabledFocusMode(baseEnvironment.urlParams.get('focusMode'))
+        .withCustomError(baseEnvironment.urlParams.get('customError'));
 
     console.log(settings);
 
+    let initialYouTubeError = null;
+    const ytErrorParam = /** @type {YouTubeError} */ (baseEnvironment.urlParams.get('youtubeError'));
+    if (ytErrorParam && YOUTUBE_ERROR_IDS.includes(ytErrorParam)) {
+        initialYouTubeError = ytErrorParam;
+    }
```

Comment:
I'd move this logic into `YouTubeErrorProvider` - pass in the urlParams if you need to, but this logic for the initial state belongs next to the feature.